### DISCUSSION TITLE
Status of running wasmer in kernel?

### DISCUSSION THREAD

**Discussion by Escapingbug** [wasmerio/wasmer#2624] [Q&A]
### Summary

The "--loader" is now missing, what is the status of running wasmer in kernel? Is the previous one still appliable? What should be changed to successfully run the previous kernel-wasm project demo?

### Additional details

Running wasmer in kernel is a promising tech that my company seems to be interested in. I see previous project but seems many things have just changed (at least the "--loader" is now missing). So what is the status of running wasmer in kernel now? Is it still possible? If it is possible, we will be happy to help on this as we are tring to achieving something like "ebpf".

**Comment by Hywan** [comment]
We should update https://github.com/wasmerio/kernel-wasm and this repository to match each other again.
